hazelcast/src/main/java/com/hazelcast/internal/serialization/SerializationServiceBuilder.java

```diff
@@ -63,5 +64,7 @@
 
     SerializationServiceBuilder setInitialOutputBufferSize(int initialOutputBufferSize);
 
+    SerializationServiceBuilder setClusterService(ClusterService clusterService);
```

Comment:
Aha, and some serializers may actually not need it at all, so that was the reason why I didn't change the `StreamSerializer` API.